### DISCUSSION TITLE
İbax Cüzdan

### DISCUSSION THREAD

**Discussion by orkunbaris** [IBAX-io/go-ibax#157] [Ideas]
İbax in kendi cüzdanını ayrı bir platformda olustursaniz daha iyi olmazmi metamask vb. Gibi

**Comment by Grekny** [comment]
bence çok mantıksız zaten kendi ağı olduğu için ilerde metamaskına da atabiliceksin muhtemelen böyle geleceği olan bir coin için kendine ait walletı olmazsa olmaz
